### DISCUSSION TITLE
Selective recording feature?

### DISCUSSION THREAD

**Discussion by Laink** [obsproject/obs-studio#11643] [General]
Is there any reason why Selective recording feature isn't a thing in OBS Studio? What I mean is being able for each source to select whether it is enabled on stream and/or on record.

This feature has been in Streamlabs Desktop for years now and it is the only thing keeping me from switching to OBS Studio. You just have to toggle a button next to each source. It is insanely useful for streamers who want to record VODs with clean overlays but might want to show sponsors/info exclusively on stream.

Thanks!

**Comment by mihawk90** [comment]
You might want to take a look at the source record plugin.
Streamlabs is intended as a one-stop-shop whereas OBS is a lot more flexible with whatever plugins you want to add.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

https://ideas.obsproject.com/posts/41/multiple-video-outputs-selective-recording-iso-recording

Thank you!
